docs/source/support/release-notes.rst

```diff
@@ -119,6 +126,13 @@ DAML-LF
 
 - Add new version 1.5. See `DAML-LF 1 specification <https://github.com/digital-asset/daml/blob/master/daml-lf/spec/daml-lf-1.rst#version-1-5>`_ for details.
 
+LEDGER
+~~~~~~
+
+- **BREAKING CHANGE**: The string fields `application_id`, `command_id`, `ledger_id`, and `workflow_id`
```

Comment:
In order to understand why this was a breaking change, we need to know what changed: what was the situation before? Did these fields have to match a different expression?

Reply:
Is it better now ?

Reply:
yep, great, thanks!